v-next/hardhat/test/fixture-projects/flatten-task/consistent-build-info-names/hardhat.config.js

```diff
@@ -0,0 +1,7 @@
+// This project is compiled from scratch multiple times in the same test, which
+// produces a lot of logs. We override this task to omit those logs.
```

Comment:
In the v3 implementation of the task, we don't do the compilation so I don't think we need this either. In fact, I think we should be able to remove all the `hardhat.config.js`/`hardhat.config.ts` from these projects and still get the tests to work as expected. To be verified though.

Reply:
That's correct. I just removed all of the config files